### PR TITLE
Added support for Quanta LB9 rack switch and updated logo to the newer QCT logo

### PR DESCRIPTION
**Pull request by stokbaek**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.
- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

**Comment by laf** [comment]
Thanks, this also needs OS discovery unit tests adding: http://docs.librenms.org/Developing/Support-New-OS/#os-test-units

**Comment by laf** [comment]
This is good to go but you've not signed the contributors agreement. You can push another commit with this as per the link you've checked at the top.

**Comment by laf** [comment]
@stokbaek close but a couple of issues, you need to rebase as the AUTHORS.md file you have is old. The commit message also has to be exactly what's stated in the document.

It's best practice not to submit pull requests from master, we have docs on best practice you can read but for now if you fix the above all should be well.

**Comment by stokbaek** [comment]
@laf sorry for all this. Guess it went abit too fast for me sadly, all should be done now.

**Comment by laf** [comment]
All good now just needs another rebase.

File: tests/snmpsim/quanta-lb9.snmprec [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|LB9, Runtime Code
```
Comment (murrant) [inline tests/snmpsim/quanta-lb9.snmprec]: Do you have the full string for sysDescr?  Or is this it?

Comment (stokbaek) [inline tests/snmpsim/quanta-lb9.snmprec]: I didn't include the full length as it is some version numbers.
Full length: ```LB9, Runtime Code 1.4.18.00, Linux 2.6.35```

**Pull request merged**